AutomatticTracks/src/main/java/com/automattic/android/tracks/MessageBuilder.java

```diff
@@ -18,7 +18,9 @@
     private static final String EVENT_TIMESTAMP_KEY = "_ts";
     private static final String REQUEST_TIMESTAMP_KEY = "_rt";
     private static final String USER_TYPE_KEY = "_ut";
-    private static final String USER_TYPE_ANON= "anon";
+    private static final String USER_TYPE_ANON = "anon";
+    private static final String USER_TYPE_WPCOM = "wpcom:user:id";
+    private static final String USER_TYPE_SIMPLENOTE = "simplenote";
```

Comment:
Corrected in https://github.com/Automattic/Automattic-Tracks-Android/commit/743e22d0c7e3eca09a04c45a01316c7cdaa9ce39